app/src/main/java/com/pwhs/quickmem/presentation/app/search_result/SearchResultUiEvent.kt

```
@@ -1,3 +1,5 @@
 package com.pwhs.quickmem.presentation.app.search_result
 
-sealed class SearchResultUiEvent()
+sealed class SearchResultUiEvent(){
```

detekt

[warning] An empty default constructor can be removed.

(detekt.empty-blocks.EmptyDefaultConstructor)